evap/staff/tests/test_views.py

```diff
@@ -925,19 +976,24 @@ class TestLoginKeyExportView(WebTest):
 
     @classmethod
     def setUpTestData(cls):
-        baker.make(UserProfile, email='manager@institution.example.com', groups=[Group.objects.get(name='Manager')])
-
+        cls.manager = make_manager()
         cls.external_user = baker.make(UserProfile, email="user@external.com")
         cls.internal_user = baker.make(UserProfile, email="user@institution.example.com")
 
         semester = baker.make(Semester, pk=1)
-        baker.make(Evaluation, pk=1, course=baker.make(Course, semester=semester), participants=[cls.external_user, cls.internal_user], voters=[cls.external_user, cls.internal_user])
+        baker.make(
+            Evaluation,
+            pk=1,
+            course__semester=semester,
+            participants=[cls.external_user, cls.internal_user],
+            voters=[cls.external_user, cls.internal_user]
```

Comment:
```suggestion
            voters=[cls.external_user, cls.internal_user],
```
(Trailing comma is also missing in more occurrences)